DepotDownloader/ContentDownloader.cs

```diff
@@ -1295,5 +1281,48 @@ private static async Task DownloadSteam3Async(uint appId, List<DepotDownloadInfo
             }
 
         }
+
+        static void DumpManifestToTextFile( DepotDownloadInfo depot, ProtoManifest manifest )
+        {
+            var txtManifest = Path.Combine( depot.installDir, $"manifest_{depot.id}_{depot.manifestId}.txt" );
+            var sw = new StreamWriter( txtManifest );

```

Comment:
`StreamWriter` is disposable so use `using var` here if we're using a new-enough version of C#, otherwise `using (var..) { ... }`. Remove the `sw.Close()` call as disposal will handle that.